extension/event-page.js

```diff
@@ -5,6 +5,25 @@
 // subject to an additional IP rights grant found at
 // http://polymer.github.io/PATENTS.txt
 
+let arc;
```

Comment:
It seems a little problematic to define these as globals with an asychronous init. The functions below assume these have already been initialized by the time they execute.

Perhaps these could be returned by `init` and the result `await`ed by the various handlers.

Reply:
I like it. I'll file an issue for this as well, so it's a more discrete refactor. #444